SDDC_FX3/USBhandler.c

```diff
@@ -134,9 +135,8 @@ void r820_initialize(uint32_t freq)
 
 	tuner.cfg = &tuner_config;
 
-	uint32_t bw;
 	r82xx_init(&tuner);
-	r82xx_set_bandwidth(&tuner, 8*1000*1000, 0, &bw, 1);
+	r82xx_set_bandwidth(&tuner, bw, 0, &bw, 1);
```

Comment:
i prefer this be a separate PR for both interface change and firmware change. Also change firmware needs to upload new firmware binary.

Reply:
The firmware binary is built during compilation of libsddc and included into it as a BLOB now. I think, it may be included into windows library this way too. I have not done this as I can't build Extio_SDDC...
Take a look at the original commit sequence and suggest changes (commits?), that may be included into a separate PR, please.
https://github.com/vladisslav2011/ExtIO_sddc/tree/linux_fixes

Reply:
it is streamlined to build on Linux easily. For Windows development, I normally don't recompile firmware for every build. :(